### ISSUE TITLE
add support for named import zod usage

### ISSUE THREAD

**Issue by marcalexiei** [marcalexiei/eslint-plugin-zod-x#109]
### 🧩 Description

Add support for named exports

```ts
import { any } from 'zod'

const Schema = any() // this should trigger a linting error, even if it can't be autofixed
```

> [!CAUTION]
> Some of the rules might require to update import node so they won't include an auto fix option

<details><summary>Details</summary>
<p>

```ts

import dedent from 'dedent';
import { RuleTester } from '@typescript-eslint/rule-tester';

import { noAny } from './no-any.js';

const ruleTester = new RuleTester();

ruleTester.run('no-z-any', noAny, {
  valid: [
    {
      name: 'with another zod schema',
      code: dedent`
        import * as z from 'zod';
        const schema = z.string();
      `,
    },
    {
      code: dedent`
        import * as z from 'zod';
        const schema = z.object({ name: z.string() });
      `,
    },
    {
      name: 'not zod',
      code: 'something.any()',
    },
  ],
  invalid: [
    {
      name: 'namespace import',
      code: dedent`
        import * as z from 'zod';
        const schema = z.any();
      `,
      errors: [
        {
          messageId: 'noZAny',
          suggestions: [
            {
              messageId: 'useUnknown',
              output: dedent`
                import * as z from 'zod';
                const schema = z.unknown();
              `,
            },
          ],
        },
      ],
    },
    {
      name: 'named import',
      code: dedent`
        import { any } from 'zod';
        const schema = any();
      `,
      errors: [{ messageId: 'noZAny' }],
    },
    {
      name: 'namespace import within an object',
      code: dedent`
        import * as z from 'zod';
        const schema = z.object({ prop: z.any() });
      `,
      errors: [
        {
          messageId: 'noZAny',
          suggestions: [
            {
              messageId: 'useUnknown',
              output: dedent`
                import * as z from 'zod';
                const schema = z.object({ prop: z.unknown() });
              `,
            },
          ],
        },
      ],
    },
  ],
});

```

</p>
</details> 

### ✒️ Additional context

- #108

**Comment by marcalexiei** [comment]
Fix available in [v1.10.0](https://github.com/marcalexiei/eslint-plugin-zod-x/releases/tag/v1.10.0).
